scripts/release.sh

```diff
@@ -133,6 +133,9 @@ function advance_admiral() {
 function advance_projects() {
     write_component "submariner-operator"
     write_component "submariner-charts"
+    if [[ -z "${semver['pre']}" ]]; then
+        set_bundle
+    fi
```

Comment:
```suggestion
    [[ -n "${semver['pre']}" ]] || set_bundle
```

Also this should be inside `advance_admiral` (I know it's a bit confusing, we can fix that in another PR)

Reply:
I left a TODO for your comment